file2http.go

```diff
@@ -81,8 +81,9 @@ func main() {
 	} else if len(*get) > 0 {
 		if strings.Count(*get, "%s") != 1 {
 			log.Fatal("Invalid get address - must be a format string")
+		} else {
+			publisher = &GetPublisher{PublisherInfo{*get}}
```

Comment:
Because the `if` block contains a terminal condition (`log.Fatal`) it's typical Go style to avoid the indentation and wrapping of the unnecessary 'else' block.